src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+public class Tagging {
```

Comment:
Missing header comment. All non-trivial classes/ methods should have javadoc format header comments.